ui/src/app/job-list/job-list.component.ts

```diff
@@ -75,6 +81,18 @@ export class JobListComponent implements OnInit {
         this.handleError(event.error);
       }
     });
+
+    // set project ID (if any) and get display field info for list columns
+    const req = URLSearchParamsUtils.unpackURLSearchParams(this.route.snapshot.queryParams['q']);
+    this.projectId = req.extensions.projectId || '';
+    const savedColumnSettings = this.settingsService.getDisplayColumns(this.projectId);
+    let field: DisplayField;
+    this.capabilities.displayFields.forEach((df) => {
+      field = df;
```

Comment:
I don't think there's any reason to do this assignment. You're still pointing to the same value (by reference) as df, so you may as well use df.